.github/workflows/Perf_Env_Build_Test_CI.yml

```diff
@@ -255,10 +255,8 @@ jobs:
         run: |
           pip install setuptools
           version=$(python3 setup.py --version)
-          sudo docker build --build-arg VERSION=$version -t ${{ secrets.QUAY_PRIVATE_BENCHMARK_RUNNER_REPOSITORY }}:v$version .
```

Comment:
There should be a versioned name as well as latest, so it's easy to back off to an earlier version if needed.

Reply:
Agree, keep it